site/src/docs/running.md

```diff
@@ -187,6 +187,9 @@ cargo nextest run -E 'platform(host)'
 `--no-fail-fast`
 : Do not exit the test run on the first failure. Most useful for CI scenarios.
 
+`--max-fail`
+: Number of tests that can fail before aborting the test run. Useful for uncovering multiple issues without having to run the whole test suite. Mutually exclusive with `--no-fail-fast`
```

Comment:
A few suggest changes here:

1. Move `--max-fail` to above `--no-fail-fast`, and write it out as `--max-fail=N`.
2. I'd just drop the "mutually exclusive" bit, it's something people can discover for themselves :) Instead, to `--no-fail-fast` you can add something like "Equivalent to `--max-fail=1`."
3. Could you tag this with `<!-- md:version 0.9.86 -->` like `--run-ignored only` below? (We should have a tag for "unreleased" at some point but just saying the next version is okay for now).